notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue

```diff
@@ -40,9 +40,8 @@ export default {
   }),
   created() {
     this.$notesService.getNoteById(this.id).then(note => {
-      const noteSpace = note.wikiOwner.split('/')[2];
       this.noteTitle = note.title;
-      this.noteUrl = `${eXo.env.portal.context}/g/:spaces:${noteSpace}/${noteSpace}/notes/${note.id}`;
+      this.noteUrl = note.url;
```

Comment:
may be changing `noteUrl` to a computed value, WDYT ?

Reply:
Done!